### ISSUE TITLE
DL3005 Do not use apt-get upgrade or dist-upgrade - Doesn't care about dist-upgrade

### ISSUE THREAD

**Issue by 06kellyjac** [hadolint/hadolint#514]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior

`DL3005` should fail if there is `dist-upgrade`

### Actual behavior

Doesn't fail

### Steps to reproduce the behavior

```Dockerfile
FROM debian:buster
RUN dist-upgrade

```

https://github.com/hadolint/hadolint/blob/master/src/Hadolint/Rules.hs#L429
